test/e2e/framework.go

```diff
@@ -134,6 +134,8 @@ const (
 	statefulSetRestartAnnotationKey = "antrea-e2e/restartedAt"
 )
 
+var testNamespace string
```

Comment:
I think it shouldn't continue using a global variable for testing namespace, which makes it impossible to run the tests in parallel in the future. Each test should use local variable for its own test namespace.

Reply:
okay will make the necessary changes.